bin/segment2.c

```diff
@@ -2653,7 +2653,7 @@ const Gfx dl_billboard_num_F[] = {
 
 #ifdef HD_SHADOWS
 ALIGNED8 static const Texture texture_shadow_quarter_circle_64[] = {
-#include "textures/segment2/shadow_quarter_circle_64.ia8.inc.c"
+#include "textures/segment2/shadow_cozies.i8.inc.c"
```

Comment:
Not sure I understand the point of this change, especially for `HD_SHADOWS` only

Reply:
Oh I can't read